openprovider/openprovider.php

```diff
@@ -1475,16 +1501,15 @@ private function manageDomainContacts(
             $result = $this->createOrReuseDomainContactsInOp($api, $op_domain['id'], $handles, $domain_contacts_from_op, $domain_contacts_from_post);
 
             if ($result != 'success') {
-                $vars->error = $result;
+                $this->showError($result);
```

Comment:
it's strange logic 
showError
showView
maybe you need to use 
setError method
and then return false; or $this->showView?